freetextresponse/freetextresponse.py

```diff
@@ -0,0 +1,486 @@
+"""
+This is the core logic for the FreeText XBlock
```

Comment:
Is it `FreeText` or `Free-text` or some other permutation?

While I'd lean toward `Free-text`, I don't care too strongly, though I would like to see it represented consistently throughout the code and documentation.